navit/maptool/maptool.c

```diff
@@ -141,7 +143,10 @@ progress_time(void)
 	seconds%=60;
 	pos+=assafe_strcp2buf(seconds>9?":":":0", buflen-pos, buf+pos);
 	pos+=assafe_lltoa(seconds, buflen-pos, buf+pos);
-	write(2,buf,pos);
+	write_result = write(2,buf,pos);
+	if (write_result == -1){
```

Comment:
we can probably avoid the need of the write_result var and check the return directly via if ()